webapp/components/PostCard/PostCard.vue

```diff
@@ -132,7 +132,7 @@ export default {
       )
     },
     isPinned() {
-      return this.post && this.post.pinnedBy
```

Comment:
I added missing setup to `posts` factory. @mattwr18 @aonomike do we want to change implementation here?

Reply:
I thought the idea was to use `this.post.pinned` instead of pinnedBy... see line `135` 
The reason is that `pinnedBy` is an author with nested attributes, ie more expensive request.. whereas, pinned is an attribute.
The reason it was `pinnedBy` in the first place is that I hadn't full taken into account the performance on production data, then you jumped in and fixed it by adding the `pinned` attribute, then he forgot to refactor this function.